backend/app/core/localization.py

```
@@ -483,6 +483,42 @@
     "Your next step could not be loaded": "Dein nächster Schritt konnte nicht geladen werden",
     "Step {step} of 4": "Schritt {step} von 4",
+    "Comparing report-intake options...": "Wege für den Report-Eingang werden verglichen...",
+    "Your report-intake recommendation could not be loaded": "Deine Empfehlung für den Report-Eingang konnte nicht geladen werden",
+    "Recommended report intake": "Empfohlener Report-Eingang",
+    "Complexity": "Komplexität",
+    "Confidence": "Sicherheit",
+    "Already configured": "Bereits konfiguriert",
+    "Report data flow": "Datenfluss der Reports",
+    "First-report status": "Status des ersten Reports",
+    "Accepted": "Akzeptiert",
+    "Duplicates": "Duplikate",
```

Ruff (0.16.0)

[error] Dictionary key literal `"Duplicates"` repeated

Remove repeated key literal `"Duplicates"`

(F601)